.github/workflows/release.yml

```diff
@@ -135,6 +135,19 @@ jobs:
           # is true. Use the empry string instead.
           BABEL_8_BREAKING: ${{ needs.check-release-type.outputs.is-babel-8 == 'true' || '' }}
 
+      - name: Generate babel-types docs
+        continue-on-error: true
+        run: |
+          node ./packages/babel-types/scripts/generators/docs.js > ./build/types.md
+
+      - name: Upload babel-types docs
+        continue-on-error: true
```

Comment:
We should run this only if the previous step didn't fail, if possible

Reply:
The `continue-on-error: true` marks that if this step fails, the release job will continue regardless. If the previous step fails, the release job should stop so this step won't be executed.